packages/payload/src/admin/components/forms/field-types/JSON/index.tsx

```diff
@@ -71,10 +70,17 @@ const JSONField: React.FC<Props> = (props) => {
   )
 
   useEffect(() => {
-    if (hasLoadedValue) return
-    setStringValue(JSON.stringify(value ? value : initialValue, null, 2))
-    setHasLoadedValue(true)
-  }, [initialValue, value])
+    const hasValue = value && value.toString().length > 0
+    if (hasLoadedValue) {
+      setStringValue(hasValue ? JSON.stringify(value, null, 2) : '')
```

Comment:
Should this be in a `try` block also?
I'm assuming you are using `try/catch` to prevent errors from parsing and stringifying JSON.

Reply:
Yes good call - updated!